src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+    <!-- chat box -->
+    <div class="chat-widget">
+        <!-- client -->
+        <div class="client">
+            <div class="chat-header">
+                <button class="header-icon">
```

Comment:
What is this button used for?

Reply:
this button will be used for resizing the widget (or can open it as a new window)

Reply:
Alright. 👍🏻